readthedocs/notifications/messages.py

```diff
@@ -302,7 +290,7 @@ def get_rendered_body(self):
                 """
             The configuration file required to build documentation is missing from your project.
             Add a configuration file to your project to make it build successfully.
-            Read more at https://docs.readthedocs.io/en/stable/config-file/v2.html
+            Read more in our <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">config file documentation</a>.
```

Comment:
For consistency, `configuration`:

```suggestion
            Read more in our <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">configuration file documentation</a>.
```